advisories/a90146.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Technical Advisory 90146
+advisory: A-90146
+summary: Changefeeds using the <code>initial_scan='only'</code> or <code>schema_change_policy='stop'</code> options may incorrectly complete with a <code>successful</code> status under certain circumstances.
```

Comment:
I don't think we need the `code` tags in the summary, unless you've seen this successfully used elsewhere.